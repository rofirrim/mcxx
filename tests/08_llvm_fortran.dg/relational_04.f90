! <testinfo>
! test_generator="config/mercurium-fortran-llvm run"
! test_stdout="relational_04.f90.out"
! </testinfo>
PROGRAM MAIN
    IMPLICIT NONE
    LOGICAL :: L1, L2

    L1 = .TRUE.
    L2 = .TRUE.
    PRINT *, L1 .EQV. L2
    PRINT *, L1 .NEQV. L2

    L1 = .TRUE.
    L2 = .FALSE.
    PRINT *, L1 .EQV. L2
    PRINT *, L1 .NEQV. L2

    L1 = .FALSE.
    L2 = .TRUE.
    PRINT *, L1 .EQV. L2
    PRINT *, L1 .NEQV. L2

    L1 = .FALSE.
    L2 = .FALSE.
    PRINT *, L1 .EQV. L2
    PRINT *, L1 .NEQV. L2
END PROGRAM MAIN
