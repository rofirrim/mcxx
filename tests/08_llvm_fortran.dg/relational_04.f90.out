 T
 F
 F
 T
 F
 T
 T
 F
